js/main.js

```diff
@@ -30,16 +29,19 @@ function deleteBook(mybook) {
   mybook.target.offsetParent.parentElement.remove();
 }
 
+function ConfirmDelete() {
+  alert('Record will be deleted!');
+}
+
 function statusChange(status) {
   if (status.target.textContent === 'Read') {
     status.target.textContent = 'Not Read';
   } else {
     status.target.textContent = 'Read';
-  } 
+  }
 }
 
 Card.prototype.createCard = (book) => {
```

Comment:
- [ ]  Separate your dom-manipulation from the application logic.
![dom_logic_separation_requirement](https://user-images.githubusercontent.com/32361963/105152589-724e2a00-5b18-11eb-855a-b37cdd1e3bb6.png)

Reply:
Why don't we create a separate file for our DOM manipulations tasks? Currently, we have the application logic and DOM in the same file. This is what we want to prevent. Kindly fix it

Reply:
Please consider checking the linter issue because is talking about the .js extension of the file but if we delete that extension the app doesn't work properly.